services/core-api/tests/services/test_issue_to_orgbook_service.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime
+from flask import current_app
+from unittest.mock import MagicMock, patch, PropertyMock
+
+from app.api.services.issue_to_orgbook_service import OrgBookIssuerControllerService
+
+from app.api.parties.party.models.party_orgbook_entity import PartyOrgBookEntity
+
+from tests.factories import create_mine_and_permit, MinePartyAppointmentFactory
+
+
+def test_issue_vc_without_orgbook_association_does_nothing(test_client, db_session, auth_headers):
+    mine, permit = create_mine_and_permit()
+    MinePartyAppointmentFactory(permit=permit, permittee=True)
+    with current_app.test_request_context() as a, patch(
+            'app.api.services.issue_to_orgbook_service.requests.post') as mock:
```

Comment:
This module traversal was getting confused because the app module has an api attribute, when i wanted it to traverse into the api module..

I've renamed the local variable in the file, but i can change it's name or do some more research on if this can be changed.